### ISSUE TITLE
Resizing columns issues when dragging too far or too small

### ISSUE THREAD

**Issue by iKrushYou** [gregnb/mui-datatables#446]
I do apologize if this has been answered before (but I've checked and couldn't find this specific issue)

I'm noticing that on any table, with the simplest configurations, resizing columns does not work well at all.. Someone, please correct me if I am missing a configuration or something.

It would be my expectation that resizing the column larger would cause the table to overflow right or for a column to be made smaller and then to just hide the rest of the data in the cell (like excel)

https://codesandbox.io/s/o96194zn09

![resize columns muidatatable](https://user-images.githubusercontent.com/1500721/53447495-38835000-39e3-11e9-8cc0-738a4ff78e56.gif)

I tried adding an override on the overflowX but that didn't work either

```const myTheme = createMuiTheme({
  overrides: {
    MUIDataTable: {
      responsiveScroll: {
        maxHeight: "580px",
        overflowX: "scroll"
      }
    }
  }
});
```

**Comment by gregnb** [comment]
There are some other issues opened around the resize columns not working properly. I would love some help on this if you have time

**Comment by domind** [comment]
Hi, can I work on this?

**Comment by gabrielliwerant** [comment]
@domind, absolutely! I'll assign this issue to you. Thanks for your interest in chipping in!

**Comment by domind** [comment]
passed [PR](https://github.com/gregnb/mui-datatables/pull/1144) for this hope it is OK

**Comment by domind** [comment]
@gabrielliwerant Hi, tried to limit movement based on content of cells, but problem is that there is no fixed min width, so finally had to leave code with fixed minimum width.
Other approach, to have it look nicer, might be, to reset style of cells in resize mode to `overflow: hidden`.

**Comment by gabrielliwerant** [comment]
@domind I think it's probably best for you to open a new issue with a demonstration of the issue you're talking about.

**Comment by patorjk** [comment]
This has been resolved as of version 3.0.1 of the library. You can see an example here:

https://codesandbox.io/s/muidatatables-custom-toolbar-zomv5?file=/index.js

**Comment by fluxquantum** [comment]
> This has been resolved as of version 3.0.1 of the library. You can see an example here:
> 
> https://codesandbox.io/s/muidatatables-custom-toolbar-zomv5?file=/index.js

@patorjk Hello, thank you for your contributions on this library. Could you describe what the fix was? I am using version 3.7.6 and have experience a similar overlap behavior while resizing, in fact in my case there's only one column that is able to move and the rest seemed fix despite setting it otherwise in my options. May I know if there's some sort of limit or setting I should be aware of that could cause this issue? Happy to create a separate issue if the problem seems more involved.

I am using the following options.

const options = {
    filter: true,
    filterType: "dropdown",
    resizableColumns: true,
    responsive: "standard",
    download: false,
    print: false,
    draggableColumns: {enabled: true, transitionTime: 1},
    selectableRows: false,
    rowsPerPageOptions: [5,10,25]
  };
